ui/apps/pixano/src/routes/[dataset]/dataset/[itemId]/+page.svelte

```diff
@@ -79,14 +80,17 @@ License: CECILL-C
                   video.sort((a, b) => a.data.frame_index - b.data.frame_index);
                 } else {
                   item_type = "vqa";
+                  const image = view[0] as Image; //TMP ??
```

Comment:
this is a workaround because refcoco is currently wrongly built with "list[Image]" instead of plain "Image"
this should not be kept IMO
(-> put view[0] in view )